src/Streamly/Tutorial.hs

```diff
@@ -1263,7 +1263,7 @@ import Control.Monad.Trans.Class   (MonadTrans (lift))
 -- main = (S.'toList' $ fmap show $ S.'fromFoldable' [1..10]) >>= print
 -- @
 --
--- Also see the 'mapM' and 'sequence' functions for mapping actions, in the
+-- Also see the 'mapM' and 'sequence' functions for mapping actions concurrently, in the
```

Comment:
This isn't always true. For serial streams, the actions are run sequentially.

Reply:
The statement does not say that you can map _only_ concurrently, it is to say that you _can_ map concurrently too using those. We can perhaps rephrase it something like this:

```
'fmap' always maps serially, see 'mapM' which can map concurrently.
```

And we can remove "it is always serial" from the previous paragraph.